.github/workflows/trigger_preview.yml

```
@@ -0,0 +1,23 @@
+name: "Trigger Preview Update"
+
+on:
+  pull_request_target:
+    types:
+      - opened
+      - synchronize
+      - closed
+      - reopened
+  push:
+    branches:
+      - main
+      - project-*
+
+jobs:
+  trigger:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Trigger Bot
```

（非常に細かいですが）

Botが動くのは向こうの設計なので、こっちはトリガーだけでいいかも。
```suggestion
      - name: Trigger
```